### ISSUE TITLE
[BUG] Alpha does not work properly when drawing to canvases

### ISSUE THREAD

**Issue by Nehemek** [lovebrew/LovePotion#135]
### **Describe the bug**
When setting an fractional alpha value with love.graphics.setColor() and drawing to a canvas, elements will be drawn with a wrong alpha value.

### **Expected behavior**
Alpha should work on canvases the same way it works when drawing normally.

### **Relevant Code To Reproduce**
The following code will produce the screenshot below:
```
function love.load()
    love.graphics.setBackgroundColor(0.8,0.75,1)
    love.graphics.set3D(false)
    canvas = love.graphics.newCanvas(200,240)
end

function love.draw(screen)
    if screen == "top" then
        love.graphics.setColor(1,1,1,1)
        love.graphics.circle("fill",100,60,16)
        love.graphics.setColor(1,1,1,.5)
        love.graphics.circle("fill",100,120,16)
        love.graphics.setColor(1,1,1,.2)
        love.graphics.circle("fill",100,180,16)

        love.graphics.setCanvas(canvas)
        love.graphics.clear(255,255,255,0)
        love.graphics.setColor(1,1,1,1)
        love.graphics.circle("fill",100,60,16)
        love.graphics.setColor(1,1,1,.5)
        love.graphics.circle("fill",100,120,16)
        love.graphics.setColor(1,1,1,.2)
        love.graphics.circle("fill",100,180,16)
        love.graphics.setCanvas()

        love.graphics.setColor(1,1,1,1)
        love.graphics.draw(canvas,200,0)
    end
end

function love.gamepadpressed(joystick, button)
    if button == "start" then love.event.quit() end
end
```

### **[Screenshots]**
![canvasalphaissue](https://user-images.githubusercontent.com/66986051/115944900-5abb7000-a47e-11eb-9e23-b2e7420a5d00.png)

### **System Info [Please complete the following information]:**
System: 3ds v11.14.0-46U
CFW: Luma v10.2.1
Method of execution: Homebrew App
LÖVE Potion version: 2.0.0

**Comment by TurtleP** [comment]
How did you test this originally? Someone tested this on hardware (aside from me) and the circles appear to be fine.

**Comment by Nehemek** [comment]
Well, I took the screenshot minutes before submitting and did so in a o3DS. Also I'm sure it has been present in all the dev versions since the canvas fix (that's when I first noticed the issue). The code I ran is the exact one above and this also happens in my main project.

**Comment by TurtleP** [comment]
Fixed with commit 7892bfd
